ios/FluentUI/Vnext/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+// MARK: - State
+
+public class PopupMenuItemCellState: MSFListCellState {
+
+	// Set/Unset selection for item cell
+	@objc @Published public var isSelected: Bool = false {
+		didSet {
+			if isSelected {
```

Comment:
Hmmmm.... 
I get why you're doing this here in the object state, but I think the right place to set these would be in the SwiftUI level based on the published isSelected property.

Thoughts?

Reply:
The current design uses state interface to update the underlying` ListCell`. To accomplish the same thing in SwiftUI it would need to have modifiers. Is there any alternative approach you were recommending?

Reply:
The alternative approach I thought of is to have the **isSelected** property supported by the list cell and just override the tokens of it in the "PopupMenuItemCell". We can discuss it offline before making a move on the PR.